projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -170,8 +170,14 @@ const VideoPressEdit = CoreVideoEdit =>
 			}
 
 			this.setState( { isFetchingMedia: true } );
-			const media = await apiFetch( { path: `/wp/v2/media/${ id }` } );
-			this.setState( { isFetchingMedia: false } );
+			const media = await apiFetch( { path: `/wp/v2/media/${ id }` } )
+				.catch( () => {
+					this.setState( { fallback: true } );
```

Comment:
An alternative approach would be to call `this.fallbackToCore();`, which sets the `fallback` state but also clears the `guid`. For clarity, I decided not to do this because when the post is saved in that state, the 'placeholder' will also render on the frontend. This causes some potentially confusing/inconsistent behavior:

1. User publishes a post with a video block.
2. User deletes the video file from their media library.
3. User inspects the published post and sees the video displays a broken link.
4. User opens the post in the editor, does nothing, but hits 'update'.
5. User inspects the published post and sees the 'placeholder' on the frontend.

The current implementation will not affect the frontend at all.